lib/Conversion/ExportVerilog/ExportVerilog.cpp

```diff
@@ -4990,26 +4991,14 @@ void SharedEmitterState::emitOps(EmissionList &thingsToEmit, raw_ostream &os,
   }
 }
 
-/// Prepare the given MLIR module for emission.
-static void prepareForEmission(ModuleOp module,
-                               const LoweringOptions &options) {
-  SmallVector<HWModuleOp> modulesToPrepare;
-  module.walk([&](HWModuleOp op) { modulesToPrepare.push_back(op); });
-  parallelForEach(module->getContext(), modulesToPrepare,
-                  [&](auto op) { prepareHWModule(op, options); });
-}
-
 //===----------------------------------------------------------------------===//
 // Unified Emitter
 //===----------------------------------------------------------------------===//
 
 LogicalResult circt::exportVerilog(ModuleOp module, llvm::raw_ostream &os) {
-  // Prepare the ops in the module for emission and legalize the names that will
-  // end up in the output.
-  LoweringOptions options(module);
-  prepareForEmission(module, options);
   GlobalNameTable globalNames = legalizeGlobalNames(module);
 
+  LoweringOptions options(module);
```

Comment:
This changes the behavior of `exportVerilog` which is exposed through the C API (and used by python), similarly with `exportSplitVerilog`.

Which I'm guessing is why there are the CI issues-- those paths no longer run `prepareForEmission`/`prepareHWModule` when they used to.

If you fix that, does the CI issue go away? :crossed_fingers: 

(Unconfirmed, my unified/python build is rebuilding... :smile:)